packages/eclipse/src/styles/fonts.css

```
@@ -1,41 +1,30 @@
+    font-style: italic;
+    font-optical-sizing: auto;
 }
 
-@font-face {
-    font-family: "Mona Sans";
-    src: url("../static/fonts/MonaSans-ExtraBold.ttf") format("truetype");
-    font-weight: 800;
-    font-style: normal;
-    font-display: swap;
+html {
+    font-family: "Mona Sans VF";
```

Stylelint (17.3.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)